superset/assets/src/messageToasts/components/Toast.jsx

```diff
@@ -96,7 +97,7 @@ class Toast extends React.Component {
           toastType === DANGER_TOAST && 'toast--danger',
         )}
       >
-        {text}
+        <div dangerouslySetInnerHTML={{ __html: dompurify.sanitize(text) }} />
```

Comment:
we should really avoid dangerously setting html. Why not encode the URL in another way and then pull it out of the string? I know this is just to make `flash` work since it only accepts a string. But instead i'd send a flash something like:
```
Preview then save your chart using the new time range endpoints $$$SUPER_UNIQUE_TOKEN_FOR_ENCODING_LINK_FOR_HERE$$$http://superset.../
```

Then you could just do in a helper function:
```jsx
const parts = str.split('$$$SUPER_UNIQUE_TOKEN_FOR_ENCODING_LINK_FOR_HERE$$$');
if (parts.length > 1) {
  return <div>{parts.length[0]} <a href={parts.length[1]}>here</a>.</div>;
} else {
  return <div>{parts.length[0]}</div>;
}
```

Reply:
But in general, this is super hacky.... maybe there's a way to do this without needing to use `flash`?

Reply:
@williaster thoughts?